packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.controller.js

```diff
@@ -44,7 +44,10 @@ export default class DedicatedServerDashboard {
     this.RECOMMIT_IMPRESSION_TRACKING_DATA = RECOMMIT_IMPRESSION_TRACKING_DATA;
 
     this.servicesStateLinks = {
-      weathermap: WEATHERMAP_URL[this.user.country],
+      weathermap:
+        WEATHERMAP_URL[
+          this.user.country !== 'US' ? 'OTHERS' : this.user.country
+        ],
```

Comment:
**suggestion:** Simple `or` would be more readable I guess.
```suggestion
      weathermap:
        WEATHERMAP_URL[this.user.country] || WEATHERMAP_URL['OTHERS'],
```

Reply:
Done